support-frontend/assets/pages/paper-subscription-checkout/paperSubscriptionCheckoutReducer.js

```diff
@@ -205,7 +215,27 @@ export type FormActionCreators = typeof formActionCreators;
 
 // ----- Reducer ----- //
 
-function initReducer(initialCountry: IsoCountry) {
+const getInitialProduct = (productInUrl: ?string, fulfillmentInUrl: ?string): Product => ({
+  productOption:
+    productInUrl === 'Saturday' ||
```

Comment:
@rupertbates i could not find a way to make `flow` let me use the constants here but if you can find a way for it use them i'm happy to use them, I prefer them